testing/py3-fs/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
```

Comment:
Looks like that git/GitHub is not able to detect that the file was moved.

Reply:
Yeah, I have experienced that before. Seems to be a bit flaky.